src/__tests__/Asset-test.ts

```diff
@@ -42,46 +42,46 @@ describe("Assets", function () {
   });
 
   it("Adding an asset", async function () {
-    const newAsset = await testDs.uploadAsset("./README.md", this.test?.title);
+    const newAsset = await testDs.uploadAsset("./README.md", { mode: "throw-if-exists", name: this.test?.title });
```

Comment:
@dionenak These shouldn't be necessary with the interface changes I mentioned before.